src/app/home/home.component.html

```diff
@@ -121,7 +147,7 @@ <h5 class="card-title">
             </div>
         </div>
     </div>
-    <div id="outputContainer" class="outputTexts mt-5">
```

Comment:
Think `.outputTexts` was never used @Kledis98? Let me know if you know.

Reply:
Right.ID is the one that is used.The class can be dropped.